docs/values.md

```diff
@@ -226,6 +226,20 @@ The generated authtoken from the Avi controller can be specified with this flag
 and maintained by secret object. The token refresh is managed by AKO. In case of token refresh failure, a new token needs to be generated and updated
 into the secret object.
 
+To encode a token generated from controller to directly patch `avi-secret`, use
```

Comment:
Our intention here is to give few ways to encode token properly.  Should we rephrase it as: `Few ways, to encode controller generated token properly, are 
1. shell
2. Python` ?

Reply:
yes, rephrased the sentence